tests/e2e/e2e_test.go

```diff
@@ -45,3 +46,212 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 		s.Require().NotEmpty(ibcStakeDenom)
 	})
 }
+
+func (s *IntegrationTestSuite) TestStaking() {
+	s.Run("delegate_staking", func() {
+		chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+		validatorA := s.chainA.validators[0]
+		validatorAAddr := validatorA.keyInfo.GetAddress()
+		valOperA := sdk.ValAddress(validatorAAddr)
+
+		alice := s.chainA.accounts[0].keyInfo.GetAddress()
+
+		delegationAmount, _ := sdk.NewIntFromString("5000000")
+		delegation := sdk.NewCoin(uctkDenom, delegationAmount)
+
+		// Alice delegate uatom to Validator A
+		s.executeDelegate(s.chainA, 0, delegation.String(), valOperA.String(), alice.String(), feesAmountCoin.String())
+
+		// Validate delegation successful
+		s.Require().Eventually(
+			func() bool {
+				res, err := queryDelegation(chainAAPIEndpoint, valOperA.String(), alice.String())
+				amt := res.GetDelegationResponse().GetDelegation().GetShares()
+				s.Require().NoError(err)
+
+				return amt.Equal(sdk.NewDecFromInt(delegationAmount))
+			},
+			20*time.Second,
+			5*time.Second,
+		)
+	})
+}
+
+func (s *IntegrationTestSuite) TestGoverment() {
```

Comment:
typo

Reply:
Change to `TestGovernment` on latest commit